pkg/logutil/logutil.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2021 The Kubernetes Authors.
```

Comment:
This file is unnecessary--please remove it.

Reply:
This logutil is necessary if we decide to keep yq.

If we alternatively decide to import ajson (for it's JSONPath), it uses the built-in go logger, which might need a different logutil wrapper.

Reply:
The ajson library only logs in the main file, which we're not using. So no log translation is needed now. Removed.